src/solvers/solvers.jl

```diff
@@ -1,6 +1,6 @@
-include("dg_tree/dg.jl")
-include("dg_curved/dg.jl")
-include("dg_unstructured_quad/dg.jl")
-include("dg_p4est/dg.jl")
+include("dgsem_tree/dg.jl")
+include("dgsem_structured/dg.jl")
+include("dgsem_unstructured/dg.jl")
+include("dgsem_p4est/dg.jl")
 include("dg_common.jl")
```

Comment:
Is this still aptly named `dg_common.jl` or should it be `dgsem_common.jl`?

Reply:
I don't want to open this can of worms right now since it will lead to quite a lot of discussion...

Reply:
:rofl: :rofl: Agreed :-)

Reply:
Since we want to have this ready for JuliaCon but the overall restructuring will take at least a week, I guess, I would like to keep this part as it is and attack it later.